scripts/generate-exports.js

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env node
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+const fs = require('fs');
+
+// This package exposes several APIs in different files. For each export we generate a separate file in the repository root
+const publicExports = ['dom', 'internal'];
+
+publicExports.forEach(exportName => {
+  fs.writeFileSync(`${exportName}.js`, `module.exports = require('./lib/${exportName}')`);
```

Comment:
Do not use commonjs exports, they cause deoptimizations 

https://webpack.js.org/plugins/module-concatenation-plugin/#optimization-bailouts


All good otherwise

Reply:
That's fair, updated. Can do it for other repos as a follow-up

Reply:
What are the other repos? I know about [browser-test-tools](https://github.com/cloudscape-design/browser-test-tools/blob/main/scripts/generate-exports.js) one, and there using commonjs makes sense, because it is NodeJS code.

We also cannot simply convert that package to ESM, because it does not work nice with Jest (which we use in many places)

Reply:
It is also test-utils: https://github.com/cloudscape-design/test-utils/blob/main/packages/core/scripts/generate-exports.js

Besides, can update collection-hooks by adding a similar script in there.